src/sparsezoo/objects/recipes.py

```diff
@@ -67,8 +77,8 @@ def default(self) -> File:
 
         # fallback to first recipe in list
         _LOGGER.warning(
-            "No default recipe {self._default_recipe_name} found, falling back to"
-            "first listed recipe"
+            f"No default recipe {self._default_recipe_name} found, falling back to "
+            f"first listed recipe {self._recipes[0].name}"
```

Comment:
Could we add an assert or Raise an exception if self._recipes is empty?

Reply:
Yup, done!